.circleci/has-functional-changes.sh

```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/env bash
 
-IGNORE_DIFF_ON="README.md CONTRIBUTING.md .gitignore .circleci/* .github/*"
+IGNORE_DIFF_ON="README.md CONTRIBUTING.md .gitignore .circleci/* .github/* tests/*"
```

Comment:
👍

Reply:
Are we sure we want to ignore `tests` in the functional change detection ?

I understand this is not convenient that that specific PR but IMHO a change in the expected behavior requires some form of documentation in the changelog.

My 2 cents.

Reply:
I feel like a "change in the expected behavior" would come with a change in the core codebase, and therefore would have to me documented in the changelog.

If there is _only_ changes in `tests`, and none in the real packages, can there really be a  change? Do you have a concrete example in mind?

Reply:
Also before migrating to CircleCi 2 et changing the conf, `tests` was already ignored.

Reply:
I thought about test refactorings and change in test accuracy.